Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyCRCCtrl.cpp

```diff
@@ -137,26 +138,46 @@ namespace AzToolsFramework
         Q_UNUSED(debugName)
     }
 
-    AZ::u32 U32CRCHandler::GetHandlerName() const
+    QWidget* U32CRCHandler::CreateGUI(QWidget* pParent)
     {
-        return AZ::Edit::UIHandlers::Crc;
+        PropertyCRCCtrl* newCtrl = aznew PropertyCRCCtrl(pParent);
+
+        auto requestWriteCall = [newCtrl](AZ::u32)
+        {
+            EBUS_EVENT(PropertyEditorGUIMessages::Bus, RequestWrite, newCtrl);
+            AzToolsFramework::PropertyEditorGUIMessages::Bus::Broadcast(&PropertyEditorGUIMessages::Bus::Handler::OnEditingFinished, newCtrl);
+        };
+
+        QObject::connect(newCtrl, &PropertyCRCCtrl::valueChanged, this, requestWriteCall);
```

Comment:
I think this is the line that should be changed to instead only trigger the write request when editing is finished on the line edit instead of each time a character changes.

Reply:
now that you mentioned that, its probably why its so hard to use.